frontend/services/image_presentation/ImagePresentation_Service.cpp

```diff
@@ -155,11 +156,13 @@ std::tuple<
 static EntryPointInitFunctions get_init_execute_resources(void* ptr) {
     if (auto module_ptr = static_cast<megamol::core::Module*>(ptr); module_ptr != nullptr) {
         if (auto view_ptr = dynamic_cast<megamol::core::view::AbstractView*>(module_ptr); view_ptr != nullptr) {
-            return EntryPointInitFunctions{
-                std::function{megamol::core::view::view_rendering_execution},
+            return EntryPointInitFunctions{std::function{megamol::core::view::view_rendering_execution},
                 std::function{megamol::core::view::view_init_rendering_state},
-                std::function{megamol::core::view::get_gl_view_runtime_resources_requests}
-            };
+                std::function{megamol::core::view::get_gl_view_runtime_resources_requests}};
+        } else if (auto gui_ptr = static_cast<megamol::frontend::GUI_Service*>(ptr); gui_ptr != nullptr) {
+            return EntryPointInitFunctions{std::function{megamol::frontend::GUI_Service::gui_rendering_execution},
+                std::function{megamol::frontend::GUI_Service::gui_init_rendering_state},
+                std::function{megamol::frontend::GUI_Service::get_gl_gui_runtime_resources_requests}};
```

Comment:
Does the `else if` get triffered only if the `void* ptr` is a `megamol::core::Module*`?